src/optimum/nvidia/hub.py

```diff
@@ -49,6 +57,7 @@
 HUB_TRTLLM_ENGINE_PATTERNS = ["**/config.json", f"**/{FILE_TRTLLM_ENGINE_PATTERN}"]
 HUB_SAFETENSORS_PATTERNS = ["config.json", "*.safetensors", SAFE_WEIGHTS_INDEX_NAME]
 LOGGER = getLogger()
+LOGGER.setLevel(level="INFO")
```

Comment:
print the meaningful logs otherwise we are blank and hanging, not very user friendly

Reply:
Why do we have to enforce INFO here? Can't we just use the level set by the user / default one?

Reply:
The default level is warning, the user usually does not provide a level. As I think e.g. [this log](https://github.com/huggingface/optimum-nvidia/blob/5ee2ff0d83dfe2b17ba5bf5bd28167ff1a34d5a7/src/optimum/nvidia/hub.py#L299-L301) should be printed I changed the default to info.